### ISSUE TITLE
Installation Instructions

### ISSUE THREAD

**Issue by kyranb** [mckelvin/cheap_flight#1]
Hey there! Would love to try this out. Do you have a list of installation instructions or system requirements. 

I've been trying to get it up and running but terminal is crashing when running ```python cheapflight/libs/mc.py```

**Comment by mckelvin** [comment]
Hi @kyranb , would you please paste the output here. Please make sure you have memcached, libmc and msgpack installed first:

    brew install memcached
    pip install libmc msgpack

And start memcached before running the demo.

Grad to see you're interested in the project. I wish it's useful to you. There's no installation instructions yet since I was the only developer. I will add it ASAP.

**Comment by kyranb** [comment]
Thanks for the help! I don't expect you to solve my problems. Just looked like a cool thing I wanted to try out :)

I thought I had memcached installed locally, however it wasn't. I've installed all the dependencies now as follows:

```
pip install -r requirements.txt
brew install memcached
pip install libmc
pip install msgpack-python
```
I'm now getting the following error. 
``` python
cheap_flight master % python demo.py PEK KUL 2016-04-27 2016-05-08
Traceback (most recent call last):
  File "demo.py", line 8, in <module>
    from cheapflight.models.rader import fetch_and_store
ImportError: cannot import name fetch_and_store 
```

```cheapflight/models/rader.py``` doesn't contain the method ```fetch_and_store```. Where you doing some refactoring on this and renamed/abstracted this method or forgot to commit something?

Once again, no stress to make documentation if it's just a personal project! This just looked like an incredibly useful program that I wanted to try out :)

**Comment by mckelvin** [comment]
You're right @kyranb , `demo.py` was broken after a refactor. I've fixed it.

```
kelvin@PANs-MacBook-Pro:~/repos/cheap_flight(master⚡) » python demo.py airasia PEK KUL 2015-10-01 2015-10-04
2015-10-01 (THU) PEK-KUL CNY: 1499
2015-10-02 (FRI) PEK-KUL CNY: 1499
2015-10-03 (SAT) PEK-KUL CNY: 1499
```

* * *

Actually `demo.py` is an old POC snippet. A preferred way to track airline fare changes is:

1. `vagrant up` and `vagrant ssh` into the cheap_flight environment
2. `cd /vagrant`
3. `./manage.py db upgrade` to create database tables
4. `./manage.py runscript cheapflight/scripts/dev/fillup_crawl_job.py` to import origin, destination, airline and date to track . The script `fillup_crawl_job.py` is where airline schedules to track are defined.
5. run `./manage.py runscript cheapflight/scripts/rader.py` every 5 minutes
6. `./manage.py runserver -h 0.0.0.0` to start a web server
7. Check <http://localhost:5000/api/> or <http://localhost:5000/api/price_history/> and you'll find the price history.

There'll be a better web UI for the procedure in the future, but basically that's idea if you're interested in how to use this. I'll re-check and add instruction to the README when I'm free.

<img width="462" alt="screenshot 2016-02-22 23 49 49" src="https://cloud.githubusercontent.com/assets/865677/13223410/0c327396-d9bf-11e5-8ba8-b1c9963cf00c.png">

Feel free to PR, suggest or file an Issue.
